### PR TITLE
Adding Styles

### PR DESCRIPTION
**Pull request by Dharshini-saravanan**

### CODE REVIEW

File: src/Profile.html [hunk 1]
```diff
@@ -9,13 +9,13 @@ <h1 style="background-color: Grey;"> Detail </h1>
         Linkedin:<a href="https://www.linkedin.com/in/dharshinisaravanan/" target="_blank">LinkedIn</a>
     </p>
     <body>
-        <h1> Profile </h1>
+        <h1 style="background-color: Grey;"> Profile </h1>
```
Comment (kmjprabhakar) [inline src/Profile.html:12]: Can we have a different colour? Blue/Teel would be more appealing.

**Review by kmjprabhakar — CHANGES_REQUESTED** [review]
Hi @Dharshini-saravanan 
Thanks for the implementation.
kindly look into the comments and do needful.

File: src/Profile.html (hunk 1 above)
Comment (Dharshini-saravanan) [reply src/Profile.html:12]: ok

**Review by kmjprabhakar — APPROVED** [review]
LGTM

**Pull request merged**